dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -147,6 +145,27 @@
 @Repository( "org.hisp.dhis.dxf2.events.event.EventStore" )
 public class JdbcEventStore implements EventStore
 {
+
+    private static final String COMMENT_QUERY =
+        "select psic.programstageinstanceid    as psic_id," +
+        "                           psinote.trackedentitycommentid as psinote_id," +
```

Comment:
Please, remove the additional spaces

Reply:
it was to make it more readable, however I'll remove it